elastic/kibana
# [Fleet] Fix saved object import missing reference

## Summary

Resolve #127664

We had some special case to retry when we have missing reference in kibana assets but we were considering that other assets were correctly imported, according to the doc

> Objects are only created when all resolvable errors are addressed, including conflicts and missing references. For information on how to resolve errors, refer to the [examples](https://www.elastic.co/guide/en/kibana/8.1/saved-objects-api-import.html#saved-objects-api-import-example).

That PR fix by including all the saved object in the retry call for missing reference.

## How to test?

You can try to install the AWS integration in the `1.13.0` version before that fix after install you should see only one dashboard in the assets tab of the integration after you should see 24.
